packages/building-rollup/README.md

```diff
@@ -287,12 +287,14 @@ export default configs.map(config => ({
 }));
 ```
 
-Make sure to prevent any compilation done by the typescript compiler `tsconfig.json`, as babel and rollup do this for you:
+#### Disable typescript compilation
+We already mentioned this above, but this is *really important*: Make sure to prevent any compilation done by the typescript compiler (`tsc`). If you use one of the options above, you put babel or rollup in charge of the compilation of typescript. In no case do you want multiple compilers to interfere with each other.
 
+You do this by setting these options in `tsconfig.json`:
 ```json
 {
   "compilerOptions": {
-    "target": "ESNEXT",
+    "target": "ESNext",
```

Comment:
In the typescript docs it mentions `ESNEXT` as the option for target. I'm not sure if it's case sensitive, do you know if this will work?